client/src/pages/VocabDrawer/VocabDrawer.js

```diff
@@ -170,112 +170,118 @@ const VocabDrawer = ({ navigation }) => {
     navigation.goBack()
   }
 
+  const [isDisabled, setIsDisabled] = useState(false)
+
   /**
    * Either updates the vocab item or creates a new vocab item
    */
```

Comment:
Amazing! Were you able to test this? Also, where is the state variable `isDisabled` actually being checked?

Reply:
Yes, it works in testing! Also, it is being checked and updated accordingly in the success callback.